Gems/Atom/Feature/Common/Assets/Shaders/Silhouette/Silhouette.azsl

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <viewsrg.srgi>
+
+#include <Atom/Features/PostProcessing/FullscreenVertex.azsli>
+#include <Atom/Features/PostProcessing/FullscreenPixelInfo.azsli>
+
+ShaderResourceGroup PassSrg : SRG_PerPass
+{
+    Sampler LinearSampler
+    {
+        MinFilter = Linear;
+        MagFilter = Linear;
+        MipFilter = Linear;
+        AddressU = Clamp;
+        AddressV = Clamp;
+        AddressW = Clamp;
+    };
+    Texture2D<float4> m_silhouettes;
+}
+
+PSOutput MainPS(VSOutput IN)
+{
+    PSOutput OUT;
+
+    const int NUM_SAMPLES = 7;
```

Comment:
As a perf improvement we should consider adding an option for 3 or 5 taps and use shader options (hooked to cvars) to trigger them.

Reply:
That's a good idea, though I think we should put that in a separate PR because I'm pretty sure Carb is going to change this code to add the outline method they currently have which I think uses a second stencil buffer to make cut-outs and maybe doesn't use this approach at all